src/Atlassian.Stash/StashClient.cs

```diff
@@ -7,9 +7,13 @@ public class StashClient
     {
         private HttpCommunicationWorker _httpWorker;
 
-        public StashClient(string baseUrl, string base64Auth = null)
+        public StashClient(string baseUrl, string base64Auth = null, bool usePersonalAccessTokenForAuthentication = false)
```

Comment:
Used a default param here so existing implementations will still work with the original method signature.